src/Components/Common/OnlineUsersSelect.tsx

```diff
@@ -1,37 +1,47 @@
-import loadable from "@loadable/component";
 import { CircularProgress } from "@material-ui/core";
-import React, {
-  useCallback,
-  useReducer,
-  useState,
-  useEffect,
-  useRef,
-} from "react";
+import React, { useCallback, useState, useEffect } from "react";
 import { useDispatch } from "react-redux";
 import { statusType, useAbortableEffect } from "../../Common/utils";
 import moment from "moment";
 import { getUserList } from "../../Redux/actions";
 import classNames from "classnames";
 
-export const OnlineUsersSelect = (props: any) => {
+type State = {
+  loading: boolean;
+  isExpanded: boolean;
+  users: Array<any>;
+  searchTerm: string;
+  searchFieldRef: React.RefObject<HTMLInputElement>;
+};
+
+type Props = {
+  selectedUser: any;
```

Comment:
while we are at it, can we ignore this any as well?

Reply:
Yeah i think it would be better to add model types one by one with fixing sentry issue. Let me add the `User` model type

Reply:
oh nice user model is already there in the codebase 😂